appveyor.yml

```diff
@@ -10,10 +10,8 @@ branches:
     - tc-31
     - sarif-v1
 
-platform: Any CPU
```

Comment:
Dropping this [recommended by AppVeyor](https://help.appveyor.com/discussions/problems/10117-dotnet-test-with-xunit-fails-because-of-the-wrongly-resolved-path) for `dotnet test` compatibility